docs/ug/cli.md

```diff
@@ -134,22 +134,22 @@ The last modified dates will be in the same timezone specified with the `--timez
 ### `--repos`, `-r`
 
 **`--repos REPO_LOCATION`**: Specifies which repositories to analyze.
-* Parameter `REPO_LOCATION`: A list of URLs or the disk location of the git repositories to analyze, separated by spaces.
+* Parameter: `REPO_LOCATION` A list of URLs or the disk location of the git repositories to analyze, separated by spaces.
 * Alias: `-r`
 * Examples:
   * `--repos https://github.com/reposense/RepoSense.git`
-  * `--repos https://github.com/reposense/RepoSense.git c:/myRepose/foo/bar`: analyzes the two specified repos (one remote, one local) and generates one report containing details of both
+  * `--repos https://github.com/reposense/RepoSense.git c:/myRepose/foo/bar`: analyzes the two specified repos (one remote, one local) and generates one report containing details of both.
 
 <box type="info" seamless>
-
-Cannot be used with `--repos`
+ 
+Cannot be used with `--repos`.
```

Comment:
Actually I am not sure about this. Is the intention to warn user not to use `--repos` with `--config` or is it trying to imply something else?

Reply:
Can open it as a different issue for discussion since that is not the main purpose of this PR, so even if we wish to change we will not be doing it in this PR. 

As far as your qsn is concerned, I think it is to state that `--config` cannot be used with `--repos`.

Reply:
Sure, I will open a separate issue regarding this.